src/components/Forms/RegisterForm.tsx

```diff
@@ -14,18 +14,18 @@ async function doRegister(
         arg: { username: string; email: string; password: string; type: 'client' | 'nutritionist' | 'consultant' };
     },
 ) {
-    const { jwt } = await clientFetcher({
+    const response = await clientFetcher<{ jwt: string }>({
         url,
         method: 'post',
         body: { username, email, password, type },
     });
 
     // this cookie part is specific to this register request
     // mustn't appear on other calls
-    setCookie('jwt-cookie', jwt, {
+    setCookie('jwt-cookie', response?.jwt, {
         path: '/',
         maxAge: 60 * 60 * 24 * 7,
-        sameSite: 'lax',
+        sameSite: 'strict',
```

Comment:
Can you explan the diff here?

Reply:
Security reasons, mostly :D 

> Lax allows the cookie to be sent on some cross-site requests, whereas Strict never allows the cookie to be sent on a cross-site request.

https://stackoverflow.com/questions/59990864/what-is-the-difference-between-samesite-lax-and-samesite-strict